docs/reference/ingest/common-log-format-example.asciidoc

```diff
@@ -37,9 +37,9 @@ image::images/ingest/ingest-pipeline-list.png[Kibana's Ingest Node Pipelines lis
 .. Set the field input to `message` and enter the following <<grok-basics,grok
 pattern>>:
 +
-[source,js]
+[source,grok]
 ----
-%{IPORHOST:client.ip} %{USER:ident} %{USER:auth} \[%{HTTPDATE:@timestamp}\] "%{WORD:verb} %{DATA:request} HTTP/%{NUMBER:httpversion}" %{NUMBER:response:int} (?:-|%{NUMBER:bytes:int}) %{QS:referrer} %{QS:user_agent}
+%{IPORHOST:source.ip} %{USER:user.id} %{USER:user.name} \[%{HTTPDATE:@timestamp}\] "%{WORD:http.request.method} %{DATA:http.request.body.content} HTTP/%{NUMBER:http.version}" %{NUMBER:http.response.status_code:int} (?:-|%{NUMBER:http.response.body.bytes:int}) %{QS:http.request.referrer} %{QS:user_agent}
```

Comment:
@ebeahan Do you mind giving me a quick spot check here?

I followed our Apache log parsing example here:
https://www.elastic.co/blog/introducing-the-elastic-common-schema

Just wanted to ensure this is right.

Reply:
Thanks for making this update!

Only one item I noted: I'd suggest using [`url.original`](https://www.elastic.co/guide/en/ecs/current/ecs-url.html#field-url-original)  instead of `http.request.body.content` to capture the request path (`/favicon.ico` is the example).

Reply:
Done! Thanks @ebeahan!